go/Makefile

```diff
@@ -43,6 +44,10 @@ lint:
 		$(error Please install golint))
 	@test -z "$$(golint ./... 2>&1 | grep -v ^vendor/ | grep -v mock/ | tee /dev/stderr)"
 
+check:
```

Comment:
Is this new since the first round of review or did I miss it somehow?

Anyway, there is `make test` in the top level , might be nice to be consistent with the naming, or possibly even recurse (not so sure about that, I think perhaps not, although a `test-go` might be handy). I think you probably do want to hook it up in the CI though, should be a simple to add a dep to the existing `ci` rule I suppose.

Reply:
I have added a bunch of commits, including this one which adds tests on qcow.

Indeed, I missed the `test` target in the top-level.  And I'm used to the GNU Coding Standards where this guy is named `check` :)

I'll rename it.

I'll leave the decision to recur to you though: currently the top level Makefile does not walk into `go/` at all.  But I'll look at the CI.

Reply:
Re recursing please could you just add after the `test-qcow` line at the top level something like:
```
test-go:
    @make -C go test
```

Up to you whether you then want to have `circle.ci` then say `make go-test` or `make -C go test`, I don't mind.